resources/views/user/dashboard.blade.php

```diff
@@ -1,6 +1,36 @@
 @include('templates.header')
+@include('templates.nav')
 
-<h1>Dashboard</h1>
-<a href="{{route('user.logout')}}">Logout</a>
+@include('templates.flash-message')
+
+<h2>My Worklogs</h2>
+<table class="table table-striped">
+    <thead>
+    <tr>
+        <th scope="col">Created At</th>
+        <th scope="col">Worklog</th>
+        <th scope="col">Updated At</th>
+    </tr>
+    </thead>
+    <tbody>
+
+    @foreach($worklogs as $worklog)
+    <tr>
+        <td>
+            <?= date('Y-m-d, h:i A', strtotime($worklog->created_at))?>
```

Comment:
for manupulating date we can use carbon or simply $worklog->created_at->format('Y-m-d, h:i A')